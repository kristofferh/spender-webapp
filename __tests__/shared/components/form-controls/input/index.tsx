import React from "react";
import renderer from "react-test-renderer";

import Input from "shared/components/form-controls/input";

describe("Create Input snapshot", () => {
  it("should render and match snapshot", () => {
    const tree = renderer
      .create(
        <Input field={{ name: "test" }} form={{ touched: {}, errors: {} }} />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
