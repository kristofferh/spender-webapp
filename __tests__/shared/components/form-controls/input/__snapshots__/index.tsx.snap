// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create Input snapshot should render and match snapshot 1`] = `
.emotion-0 {
  position: relative;
  color: inherit;
}

.emotion-4 {
  font-weight: bold;
  display: block;
}

.emotion-4:after {
  content: '';
}

.emotion-6 {
  -webkit-appearance: none;
  -moz-appearance: none;
  -ms-appearance: none;
  appearance: none;
  border-radius: 0;
  width: 100%;
  border: 0;
  border-bottom: 1px solid #888;
  padding: 5px 0;
  min-height: 40px;
  font-family: inherit;
  font-size: inherit;
  border-bottom-color: inherit;
}

.emotion-6:focus {
  outline: none;
  border-image: linear-gradient(45deg, #4A90E2, #c6aeab) 2;
}

<div
  className="emotion-0 emotion-1"
>
  <div
    className="emotion-2 emotion-3"
  >
    <label
      className="emotion-4 emotion-5"
      htmlFor="test"
    />
  </div>
  <input
    className="form-control emotion-6 emotion-7"
    id="test"
    name="test"
    onBlur={[Function]}
    onChange={[Function]}
    type="text"
  />
</div>
`;
